### PR TITLE
Self-register instances feature documentation

### PR DESCRIPTION
**Pull request by gsanchezu**
@DenesPal, 👍?

### CODE REVIEW

File: Cloud Application Manager/Getting Started/self-register-instances.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+A virtual machine deployed with self-registration will pop up in Cloud Application Manager as an imported instance automatically, belonging to the provider whose self-registration script was used.
+
+If the virtual machine belongs to a scaling group, a single instance will appear as imported in Cloud Application Manager, containing all the machines included in the scaling group.
+
+Once the instance is available in Cloud Application Manager, you can manage it as any other instance type, execute lifecycle events or edit it through the [Lifecycle Editor](../Core Concepts/lifecycle-editor.md), such as [deploying a Script Box](./deploy-script-box-to-registered-instance.md) on the instance.
+
+### Contacting Cloud Application Manager Support
+
+We’re sorry you’re having an issue in [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/). Please review the [troubleshooting tips](../Troubleshooting/troubleshooting-tips.md), or contact [Cloud Application Manager support](mailto:incident@CenturyLink.com) with details and screenshots where possible.
```
Comment (DenesPal) [inline Cloud Application Manager/Getting Started/self-register-instances.md]: I think this is the deprecated support text, perhaps you have used an old template.

Comment (gsanchezu) [reply Cloud Application Manager/Getting Started/self-register-instances.md]: There are two flavors of the support section (one for ALM and another for CAM Platform, let's say). I tweaked this one (ALM) a little bit to talk about register an instance, but it should be it. Why do you think it is deprecated?

File: Cloud Application Manager/Getting Started/self-register-instances.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* Access to the underlying cloud provider account with authorization to deploy virtual infrastructure.
+
+### Supported provider types
+
+These are the Provider types supporting self-registration of virtual machines:
+
+* Amazon AWS/EC2
+* CenturyLink Cloud
+* CenturyLink Private Cloud on VmWare Cloud Foundation
+* Google GCP/GCE
```
Comment (EfrenRey) [inline Cloud Application Manager/Getting Started/self-register-instances.md]: Support for Google is still in progress, not available yet.

File: Cloud Application Manager/Getting Started/self-register-instances.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* VmWare vCloud Director
```
Comment (EfrenRey) [inline Cloud Application Manager/Getting Started/self-register-instances.md]: Same, I think this is still in progress

Comment (gsanchezu) [reply Cloud Application Manager/Getting Started/self-register-instances.md]: Ok, I removed these two as well as CPC on vCF for now (we need to remember to add them afterwards whenever they are available).

**Pull request merged**